raiden-dapp/src/views/TransferRoute.vue

```diff
@@ -40,15 +43,47 @@ import { Token, TokenModel } from '@/model/types';
     NoChannelsDialog,
   },
   computed: {
+    ...mapState(['latestStateBackupReminder']),
     ...mapGetters(['tokens', 'channelWithBiggestCapacity', 'openChannels']),
   },
 })
 export default class TransferRoute extends Vue {
+  latestStateBackupReminder!: number;
   tokens!: TokenModel[];
+  currentTime: number = new Date().getTime();
+  oneDay: number = new Date().setHours(24);
```

Comment:
This is not part of the components state. It is a fixed constant. Please define it outside of the class and consider to name it in upper snake case like `ONE_DAY`.